### ISSUE TITLE
Combined use of fig-align and fig-caption results in LaTeX Error: Not in outer par mode

### ISSUE THREAD

**Issue by arnaudgallou** [quarto-dev/quarto-cli#1124]
### Bug description

I'm running into the following error `LaTeX Error: Not in outer par mode` when trying to render a `.qmd` as a `pdf`. The problem only occurs when using `fig-align:` together with `#| fig-cap:`. Removing one or the other option solves the issue. Below is a reproducible example.

``````
---
format: pdf
execute: 
  echo: false
  warning: false
fig-format: pdf
fig-align: center
---

```{r}
#| fig-cap: foo
plot(1)
```
``````

Probably related to the following issue https://github.com/quarto-dev/quarto-cli/issues/462#issue-1180525040.

I'm using Quarto v. 0.9.549 in RStudio v. 4.2.0, running on macOS Monterey.

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, self-contained, and reproducible example?
- [X] documented the quarto version you're running, by providing the output produced by `quarto check` in a terminal in your issue?
- [X] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [ ] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by cscheid** [comment]
Thanks for reporting! Definitely a bug on our side, we'll fix it right away. However, I'm curious: LaTeX output by default centers figures, as evidenced by removing `fig-align: center` and inspecting the `keep-tex: true` output:

    \begin{figure}
    
    {\centering \includegraphics{fig-align-bug_files/figure-pdf/unnamed-chunk-1-1.pdf}
    
    }
    
    \caption{foo}
    
    \end{figure}

Were you trying to achieve something else besides centering the figure? I just want to ensure I'm not missing any other quarto bugs.

**Comment by arnaudgallou** [comment]
Yes, I noticed LaTeX centers figures by default while debugging the problem. In fact, I initially rendered my document as html and simply didn't remove the `fig-align` option when switching to pdf. Took me some time to understand what was going on so I wanted to report the issue anyway because it will certainly happen to other people.

**Comment by jjallaire** [comment]
Should be fixed here (https://github.com/quarto-dev/quarto-cli/commit/1c5e5d72258d99a39c894a4f8d6df21a8c70710e) and available in the the v0.9.577 release (available at https://github.com/quarto-dev/quarto-cli/releases/latest)
